README.md

```diff
@@ -63,9 +63,9 @@ end
 You can disable the timestamps for SqlLite since TIMESTAMP is not supported for this database:
 
 ```crystal
-require "granite_orm/adapter/sqlite"
+require "granite/adapter/sqlite"
 
-class Comment < Granite::ORM::Base
+class Comment < Granite::Base
```

Comment:
We actually haven't done this yet.  @faustinoaq Can you take a stab at it?

Reply:
https://github.com/amberframework/granite/pull/175